### ISSUE TITLE
Remove dots from all final yeh

### ISSUE THREAD

**Issue by lateef-shaikh** [lateef-shaikh/pakistani-mushaf#1]
Replace final ي with ى

**Comment by lateef-shaikh** [comment]
Replaced Arabic Yeh with Farsi/Urdu Yeh

**Comment by abikhaleel** [comment]
Salam Alaikum. Jazakallah for all your work.

May I ask why farsi/urdu yeh is being used. Wouldn't it make more sense to use arabic yeh?

**Comment by lateef-shaikh** [comment]
> Salam Alaikum. Jazakallah for all your work.
> 
> May I ask why farsi/urdu yeh is being used. Wouldn't it make more sense to use arabic yeh?

Walekum Salaam, the reason is that the published Quran in this style and even Uthmani style (in Saudi Arabia) don't have dots in final yeh. Therefore I have used Urdu yeh in both mushaf. Earlier I was using Arabic yeh but when I checked PDF of both mushaf then I realized that dots are not put in final yeh. I didn't want to make a custom font and I want the text to look same using all fonts. (https://youtu.be/nk3wjTBluSo)
Thank you for checking out my work. Other mushafs are still in PDF in karachvi.com They would require characters from all over the Arabic block !
